cartographer/serializers/jsonapi_serializer.py

```diff
@@ -221,6 +215,13 @@ def document_with_data(self, data, version, skip_self=True):
             response["meta"] = meta
         return response
 
+    def _get_version(self, version=None):
+        if version is None:
+            version = self._flask_json_api_version()
+            if version is None:
+                version = get_default_version()
```

Comment:
Ohhh I just thought of yet a more Pythonic way of doing this:

`return version or self._flask_json_api_version() or get_default_version()`